romfs/source/fighter/dedede/param/vl.prcxml

```diff
@@ -11,7 +11,7 @@
   <list hash="param_special_s">
     <struct index="0">
       <float hash="throw_angle_max">75</float>
-      <float hash="throw_angle_min">310</float>
+      <float hash="throw_angle_min">50</float>
```

Comment:
```suggestion
      <float hash="throw_angle_min">310</float>
```

Reply:
Are we sure we want to keep the 310 angle? It's the reason why gordo can go backwards.